vividus/src/main/java/org/vividus/transformer/SortingTableTransformer.java

```diff
@@ -29,15 +32,22 @@
 
 public class SortingTableTransformer implements ExtendedTableTransformer
 {
+    private static final String ASCENDING = "ASCENDING";
+    private static final String DESCENDING = "DESCENDING";
```

Comment:
we have enum: `org.vividus.transformer.IndexingTableTransformer.Order`, it makes sense to make it regular and reuse in both transformers

Reply:
done